cli/command/image/import.go

```diff
@@ -26,9 +26,10 @@ func NewImportCommand(dockerCli command.Cli) *cobra.Command {
 	var options importOptions
 
 	cmd := &cobra.Command{
-		Use:   "import [OPTIONS] file|URL|- [REPOSITORY[:TAG]]",
-		Short: "Import the contents from a tarball to create a filesystem image",
-		Args:  cli.RequiresMinArgs(1),
+		Use: "import [OPTIONS] file|URL|- [REPOSITORY[:TAG]]",
+		Short: `Import the contents from a tar archive to create a filesystem image
+previously exported by "docker export" command.`,
```

Comment:
Thinking of this now, I think `docker import` can be used to just import a tar-bal rootfs for the container (which is not created by `docker export`), so we may need some more working here

Reply:
/cc @mstanleyjones

Reply:
Are you saying that what ever docker export create is NOT what is used for "docker import" input? Man if it is the case things are even more obscure than I thought. What then creates tar archive for "docker import"?
docker export --help
Usage:  docker export [OPTIONS] CONTAINER
Export a container's filesystem as a tar archive

Reply:
Any `tar` command or tar library could create it, it's not just `docker export`

Reply:
@thaJeztah Do you want me to ad a couple of words here ? Something like "exported by ```docker export``` or ```tar``` commands"